neuraxle/data_container.py

```diff
@@ -57,6 +66,11 @@ def __init__(self, data_inputs: Any, current_ids=None, summary_id=None, expected
         else:
             self.expected_outputs = expected_outputs
 
+        if sub_data_containers is None:
+            sub_data_containers = []
+
+        self.sub_data_containers: List[NamedDataContainerTuple] = sub_data_containers
```

Comment:
you could remove this blank line maybe

Reply:
could probably be dict, not sure

Reply:
can we fix this in a future pr this is https://github.com/Neuraxio/Neuraxle/issues/175